web/templates/task/index.html.eex

```diff
@@ -33,8 +34,13 @@
             <%= String.slice NaiveDateTime.to_iso8601(task.finish_time), 11, 5 %>
           </td>
           <td class="mdl-data-table__cell--non-numeric">
-            <%= if task.user_id do %>
-              <%= CoursePlanner.SharedView.display_user_name(task.user) %>
+            <%= task.max_volunteers %>
+          </td>
+          <td class="mdl-data-table__cell--non-numeric">
+            <%= if task.volunteers do %>
```

Comment:
is this ever falsy? as far as I remember this will be always a list, an empty list in case there is no volunteers?